apollo-federation/src/operation/mod.rs

```diff
@@ -2919,6 +2978,26 @@ impl SelectionSet {
         Ok(normalized_selections)
     }
 
+    /// Removes the @defer directive from all selections without removing that selection.
+    // PORT_NOTE: This behavior differs from the JS code, which removes anything with @defer.
+    // TODO(@TylerBloom): After the private preview and @defer being impl-ed, check if this logic
+    // should be changed to mirror the JS code.
+    fn without_defer(&mut self) {
+        // TODO: This doesn't seem like the correct way to get the directive name...
+        let Some(defer_name) = self.schema.get_directive_definition(&name!("defer")) else {
```

Comment:
This seems incorrect, but I always forget how to pull out the directive definition from a schema because the users might have renamed the directive when they imported it.

Reply:
`@defer` is a spec name, not an imported one, so it's always `@defer` and you don't have to bother with renames :)

Reply:
Excellent! Thanks